.env.example

```
@@ -11,3 +11,7 @@ GEMINI_API_KEY=your_gemini_api_key
 # notification-service에서 사용 (슬랙 알림 발송)
 # 필수: Slack Bot Token 발급 후 입력
 SLACK_TOKEN=your_slack_token
+
+# HUB
+HUB_SERVICE_NAME=hub-service
+HUB_SERVICE_PORT=hub_service_port
```

dotenv-linter (4.0.0)

[warning] [EndingBlankLine] No blank line at the end of the file

(EndingBlankLine)